app/helpers/lists_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ListsHelper
+  def image(list)
+    if list.name == "Хочу прочитать"
+      "https://letidor.ru/thumb/1280x720/smart/filters:quality(75)/imgs/2018/04/24/07/2148932/10a07f44e5278067b11bea002261c3ae310ffc62.jpg"
+    elsif list.name == "Прочитал"
```

Comment:
I don't really like constructions with 'elsif', maybe 'case' or 'if + return' is better?

Reply:
Yep, sure, I know that I MUST refactor it, I did it, because it was faster

Reply:
I'll refactor it when I moved this logic into ViewModel